tests/unit/draggable/options.js

```diff
@@ -1045,6 +1045,7 @@ QUnit.test( "revert and revertDuration", function( assert ) {
 	testHelper.move( element, 50, 50 );
 	setTimeout( function() {
 		assert.ok( $( "#draggable2" ).is( ":animated" ), "revert: true with revertDuration should animate" );
+		ready();
```

Comment:
Is this added on purpose?

Reply:
Yes, I wanted the test to wait for this as well - that's why the call `assert.async()` now takes `2` as an argument - it means `ready()` needs to be called twice before the test is marked as finished.